### PR TITLE
Add Router latency

### LINKED ISSUE #82: Provide parameter to customize the delay/latency of a mocked response
When using kakapo for prototype or tests it's important to be able to simulate the latency (also Moya provides this for their stub).
Default should be 0 :QDD:

### PR DESCRIPTION
**Pull request by MP0w**
Closes #82

### CODE REVIEW

File: Source/Router.swift [hunk 1]
```diff
@@ -187,7 +190,18 @@ public final class Router {
...
+        }
+        
+        if latency > 0 {
+            let delayTime = dispatch_time(DISPATCH_TIME_NOW, Int64(latency * Double(NSEC_PER_SEC)))
+            dispatch_after(delayTime, dispatch_get_main_queue()) {
+                didFinishLoading(server)
+            }
+        } else {
+            didFinishLoading(server)
+        }
```
Comment (joanromano) [inline Source/Router.swift:204]: Why not just using `dispatch_after` directly? Isn't the overhead with `latency = 0` negligible?

File: Tests/RouterTests.swift [hunk 2]
```diff
@@ -150,6 +156,26 @@ class RouterTests: QuickSpec {
...
+                    
+                    NSURLSession.sharedSession().dataTaskWithURL(NSURL(string: "http://www.test.com/users/1")!) { (data, response, _) in
+                        responseData = data
+                        }.resume()
+                    
+                    
+                    let startTime = CFAbsoluteTimeGetCurrent()
+                    expect(responseData).toNotEventually(beNil(), timeout: 3)
+                    let endTime = CFAbsoluteTimeGetCurrent()
+                    expect(endTime - startTime) >= 2.0
```
Comment (joanromano) [inline Tests/RouterTests.swift:176]: Can't we have a smaller range than `> 2` with ranges?

File: Tests/RouterTests.swift [hunk 3]
```diff
@@ -150,6 +156,26 @@ class RouterTests: QuickSpec {
...
+                }
+            }
```
Comment (joanromano) [inline Tests/RouterTests.swift:178]: Probably good idea to test that adding latency to a router doesn't affect other router's latency.

**Comment by joanromano** [comment]
👍

File: Tests/RouterTests.swift (hunk 2 above)
Comment (MP0w) [inline Tests/RouterTests.swift:176]: what do you mean?

**Pull request merged**